### ISSUE TITLE
.table-responsive doesn't work inside .media-body

### ISSUE THREAD

**Issue by shkarupa-alex** [twbs/bootstrap#15622]
Demo http://jsfiddle.net/vuqm0ybb/2/

**Comment by twbs-lmvtfy** [comment]
Hi @shkarupa-alex!

You appear to have posted a live example (http://fiddle.jshell.net/vuqm0ybb/1/show/light/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* [W005](https://github.com/twbs/bootlint/wiki/W005): Unable to locate jQuery, which is required for Bootstrap's JavaScript plugins to work

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by shkarupa-alex** [comment]
I've added jQuery and Bootstrap JS http://jsfiddle.net/vuqm0ybb/2/

**Comment by mdo** [comment]
This is due to the use of `overflow` on the `.media-body` as far as I can tell.

**Comment by mjdean1994** [comment]
Doesn't look like an issue with the overflow. Seems to be more to do with the media-body being styled to a width of 10000px.

**Comment by mdo** [comment]
Yeah, true that. Sadly, nothing we can do about this I think given how the media component works. Punting as a won't fix for now.
